client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -572,6 +573,12 @@ class AllDomains extends Component {
 				path: domainManagementRoot() + '?' + stringify( { filter: 'owned-by-others' } ),
 				count: filterDomainsByOwner( nonWpcomDomains, 'owned-by-others' )?.length,
 			},
+			{
+				label: translate( 'Domain only' ),
```

Comment:
I wonder if this label is understandable enough, but I don't have other suggestions 🤔

Reply:
Let's use "Parked domains"

Reply:
My initial review thought is to use "Parked domains" since that's what we use elsewhere. I know that it might also be misleading for some cases, but maybe let's go with that for now. Thoughts?

Reply:
You're right, "Parked domains" works great for me